assets/webpack.config.js

```diff
@@ -151,14 +151,14 @@ module.exports = (env, argv) => {
       new webpack.DefinePlugin({
         ENV: {
           DEVELOPMENT: isDev,
+          GA_TRACKING_ID: JSON.stringify(
```

Comment:
Thanks for the PR! Any particular reason to use a new name for the environment variable? I'm not quite sure whether this is a new value or whether we're using the old one. Also for the default - it'd be great to still have that as some sort of env variable.

Reply:
It's a new name for the old environment variable. I restored the old name in a more recent commit. I also added the variable ENV.DEFAULT_GOOGLE_ANALYTICS_ID to store the tracking code of the default property.

Reply:
If you're working with the web application locally, you can set this environment variable on your system to have the Google analytics tracking data for your local application be sent to the Google Analytics property of your choice.